omnibus/package-scripts/chef-server/postinst

```diff
@@ -11,6 +11,45 @@ function error_exit
 	exit 1
 }
 
+version=$(awk '/chef-server/{ split($2,a,"+");print a[1];exit }' /opt/opscode/version-manifest.txt)
+
+confirm () {
+  read -r -p "Do you want to continue [y/n]" response
+  case "$response" in
+        [yY][eE][sS]|[yY])
+            echo "Putting chef infra server in maint mode sometime in the future."
+            echo "create a /tmp/first_time_internal_elasticsearch_install"
+            touch /tmp/first_time_internal_elasticsearch_install
+            echo "for now continuing."
+            ;;
+        *)
+            echo "Please reinstall the older version again and reconfigure."
+            exit
+            ;;
+  esac
+}
+
+first_elasticsearch_internal_install () {
+  IFS=. read -r major minor patch <<< "$version"
```

Comment:
Can we capture this in preinst instead, so that we don't need to operator to do a rollback/reinstall of the older version? 
One concern is that they may no longer have the exact version number they previously had installed available.

Reply:
yep! done!